src/community/infix-eng-rev.ts

```diff
@@ -0,0 +1,22 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript } from "../utils";
+import type Decimal from "break_infinity.js";
+
+export class InfixEngineeringReverseNotation extends AbstractInfixNotation {
+  public readonly name = "Reverse infix engineering";
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'name' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)